affiliate/package.json

```diff
@@ -21,7 +21,9 @@
 		"eslint-plugin-react": "7.30.0"
 	},
 	"dependencies": {
+		"@wordpress/element": "^4.18.0",
```

Comment:
I'm not sure why this wasn't included before... Maybe it just cuts down on the size of the plugin. I added this without thinking too much about it to fix a warning that I thought was causing the pr to fail (but I realized that warnings don't fail in this repo).

Starting to think that it might be best to remove this for now, since I haven't been able to dig up the reasoning for this missing...

Reply:
Might be worth having @sergunich chime in here as he has more WordPress plugin experience.

Reply:
> I'm not sure why this wasn't included before
That's because `@wordpress/scripts` has a `@wordpress/babel-preset-default` as dependency
https://github.com/orgnc/wordpress-plugin/blob/ffca2768419ca39ca304fdd5dcad70ddfe1a6b60/affiliate/package-lock.json#L3448

And `@wordpress/babel-preset-default` has a `@wordpress/element` as dependency
https://github.com/orgnc/wordpress-plugin/blob/ffca2768419ca39ca304fdd5dcad70ddfe1a6b60/affiliate/package-lock.json#L3265

So, it was installed implicitly, and yes - it's better to state it as a dependency explicitly

> Maybe it just cuts down on the size of the plugin
We don't care about the size of the plugin 🙂

Reply:
Thanks for the explanation!

Reply:
Thanks @sergunich :)